projects/mno-frontend-lib/src/lib/_components/login-box/login-box.component.html

```diff
@@ -1,18 +1,36 @@
 <div class="login-box-container">
   <div class="card login-box mx-auto">
     <h5 class="card-header text-center">{{ header }}</h5>
-    <div class="card-body d-flex flex-column justify-content-between align-items-center">
+    <div *ngIf="!signupEnabled" class="card-body d-flex flex-column justify-content-between align-items-center">
       <div class="inputs">
         <input type="email" class="form-control mt-2" [(ngModel)]="email" placeholder="your@email.com" (keydown)="onKeydown($event)">
-        <input type="password" class="form-control mt-2" [(ngModel)]="password" placeholder="password" (keydown)="onKeydown($event)">
+        <input type="password" class="form-control mt-2" [(ngModel)]="password" placeholder="Password" (keydown)="onKeydown($event)">
       </div>
 
       <button class="btn btn-primary login-button" (click)="login()" [disabled]="loading">
         {{ loading ? 'Signing in...' : 'Sign in' }}
       </button>
     </div>
+
+    <div *ngIf="signupEnabled" class="card-body d-flex flex-column justify-content-between align-items-center">
+      <div class="inputs">
+        <input type="text" class="form-control mt-2" [(ngModel)]="company" placeholder="Company" (keydown)="onKeydown($event, 'signup')">
+        <input type="email" class="form-control mt-2" [(ngModel)]="email" placeholder="your@email.com" (keydown)="onKeydown($event, 'signup')">
```

Comment:
What about `(keydown)="$event.key === 13 && login()"` ? This way you can get rid of the whole switch.